pkg/watch/manager.go

```diff
@@ -256,6 +260,10 @@ func (wm *Manager) replaceWatches(r *Registrar) error {
 		}
 	}
 
+	if err := wm.metrics.reportGvkCount(int64(len(wm.watchedKinds))); err != nil {
```

Comment:
This metric should also be updated as part of add/remove watch.

Reply:
thanks! added as part of `doAddWatch` and `doRemoveWatch` also